### DISCUSSION TITLE
Laravel.com

### DISCUSSION THREAD

**Discussion by FranciscoCaldeira** [laravel/framework#46425] [General]
Hey guys. 
The algolia don't respect user choice in seeing dark mode.

![imagem](https://user-images.githubusercontent.com/45494986/224384881-9d0517ec-d387-4d57-9cc3-2bc81c979718.png)

**Comment by Sophist-UK** [comment]
I imagine the site is using Laravel Scout or Algolia Scout-Extended.

**Reply by Sophist-UK** [reply]
I have been trying to find the source for the popup HTML in these two packages and not found it so far. But I suspect that some extra `dark:` classes will need to be added to them in order to switch the pop-up to dark mode.

**Reply by Sophist-UK** [reply]
Algolia search was implemented very recently in https://github.com/laravel/laravel.com/pull/275/files . The CSS for the search box is in https://github.com/laravel/laravel.com/blob/master/resources/css/_search.css but this imports a couple of files and I think that it may well be https://github.com/algolia/docsearch/blob/main/packages/docsearch-css/src/modal.css

Anyway, it seems that there needs to be some more CSS added by Laravel or Algolia folks to support dark theme.

**Reply by FranciscoCaldeira** [reply]
> Anyway, it seems that there needs to be some more CSS added by Laravel or Algolia folks to support dark theme.

nothing more to say 👍

**Reply by Sophist-UK** [reply]
@FranciscoCaldeira Why not work out what CSS is needed and submit a PR?

**Comment by ankurk91** [comment]
@driesvints Can you look into it

**Comment by driesvints** [comment]
We’d welcome PRs for this if anyone’s willing. Not sure when I’ll have time for it myself.

**Comment by Sophist-UK** [comment]
See also https://github.com/laravel/framework/discussions/40983
